authn/swagger.yaml

```diff
@@ -54,7 +54,7 @@ paths:
             $ref: "#/definitions/Key"
         400:
           description: Failed due to malformed query parameters.
-        403:
+        401:
```

Comment:
If we include both 401 and 403 (see my previous comment), ew should update API docs accordingly.